starknet_devnet/starknet_wrapper.py

```diff
@@ -385,15 +385,20 @@ async def calculate_actual_fee(self, external_tx: InvokeFunction):
         )
 
         gas_price = state.state.block_info.gas_price
-        gas_usage = tx_fee // gas_price
+        gas_usage = tx_fee // gas_price if gas_price else 0
 
-        return {
+        result = {
             "overall_fee": tx_fee,
             "unit": "wei",
             "gas_price": gas_price,
             "gas_usage": gas_usage,
         }
 
+        if state.state.block_info.block_timestamp == 0:
+            result["warning"] = "block isn't produced"
```

Comment:
Does this extra parameter create issues when used with Starknet CLI (`starknet estimate_fee`)?

Reply:
As I was looking way to test this I found out that RPC has a bug with extracting method name:
https://github.com/Shard-Labs/starknet-devnet/pull/189/commits/22aa29bfd6e35f0544ca19af54177e1d0dd2f75d

Reply:
cli call true straknet seems to work fine too:
```starknet --gateway_url http://http://127.0.0.1:5050 --feeder_gateway_url http://127.0.0.1:5050/feeder_gateway estimate_fee --address 0x62230ea046a9a5fbc261ac77d03c8d41e5d442db2284587570ab46455fd2488 --abi ./starknet_devnet/abi.json --function mint --inputs 1786654640273905855542517570545751199272449814774211541121677632577420730552 1000000000000000000000 0 --no_wallet```
```
The estimated fee is: 0 WEI (0.000000 ETH).
Gas usage: 0
Gas price: 0 WEI
```

Reply:
This is one of the changes of PR #186:
https://github.com/Shard-Labs/starknet-devnet/pull/186/files#r926398280